src/lib.js

```diff
@@ -77,5 +77,14 @@ module.exports = {
     const monthString = month.length < 2 ? `0${month}` : month
     const dateString = date.length < 2 ? `0${date}` : date
     return [d.getFullYear(), monthString, dateString].join('-')
+  },
+
+
+ /**
+  * Returns local timezone's offset from UTC as a string +/- HH:MM, e.g. -04:00
+  */
+  getTZCode: (d) => {
```

Comment:
add a new unit test suite for this in https://github.com/diracdeltas/accountable-triage/blob/master/test/libTest.js